runtime/voxel.c

```diff
@@ -26,6 +26,13 @@ int main(int argc, char* argv[]) {
 
     fseek(fp, 0, SEEK_SET);
     fread(data, sizeof(char), size, fp);
+    
+    if (fread(data, sizeof(char), size, fp) != size) {
+        fprintf(stderr, "Error reading file content\n");
```

Comment:
Just me being a bit pedantic but I'd recommend adding an 's' to the end of the string to make 'content' plural — imo "Error reading file contents" reads a bit better. Happy to do this myself after merging tho!